src/components/CarCard/CarCard.tsx

```diff
@@ -1,25 +1,46 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./CarCard.scss";
 import rightarrow from "../../assets/rightArrow.svg";
-import { IcarData } from "../../models/ICarData";
+import { CarDetails } from "../../models/CarDetails";
+import { useDispatch, useSelector } from "react-redux";
+import { addToCompare } from "../../redux/CompareCars/reducer";
+import { NUMBEROFCARSTOCOMPARE } from "../../constants";
```

Comment:
you can use snake_case for cinstants if all caps is involved

Reply:
@pashedin sure added the changes